BeforeDayBegins.md

```diff
@@ -31,18 +31,18 @@ soon enough
 
 a day, or two,   
 a night, or three,  
-it will be  
+but not now
```

Comment:
Ewentualnie 
```suggestion
but not today
```
i wtedy śpiewając to _ay_ bliżei _i_ możemy próbowac naciągać że jakby się rymuje

Reply:
albo po prostu "not today", ale trzeba wtedy poprawić ostatni wers też